### DISCUSSION TITLE
Improve estimated travel time

### DISCUSSION THREAD

**Discussion by Jean-BaptisteC** [organicmaps/organicmaps#3750] [Ideas]
I use OM regularly and i think travel time is underestimated.

I have doing same route with different GPS:
- OM
- [Qwant Maps](https://www.qwant.com/maps/)
- Apple Plans
- Google Maps
- [GraphOpper in OSM](https://www.openstreetmap.org/directions)

Roads used can be different but start and end are the same. I have use mode Car on all GPS.

Start point:
- https://www.openstreetmap.org/node/2117635392

End point:
- https://www.openstreetmap.org/way/44419012

Results;
- OM: 4min/2,5km
- [Qwant Maps](https://www.qwant.com/maps/routes/?mode=driving&pt=true&origin=addr%3A6.961629%3B48.293732%3A11%4011_Rue_Albert_Cuny&destination=osm%3Anode%3A4398588295%40Saint-Di%C3%A9-des-Vosges&selected=0#map=14.87/48.2871994/6.9518760): 8min/2,8km
- Apple Plans: 7min/2,5km
- [Google Maps](https://www.google.fr/maps/dir/11+Rue+Albert+Cuny,+Saint-Di%C3%A9-des-Vosges/Gare+de+Saint-Di%C3%A9-des-Vosges,+Saint-Di%C3%A9-des-Vosges/@48.2879577,6.9462585,15z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x4793ee90b82c144f:0x9920a4e8a00673d4!2m2!1d6.9613007!2d48.2938981!1m5!1m1!1s0x4793eea6faf93949:0xdd0a33f6660d8683!2m2!1d6.9485737!2d48.282061!3e0): 6min/2,4km
- [GraphOpper](https://www.openstreetmap.org/directions?engine=graphhopper_car&route=48.2937%2C6.9616%3B48.2820%2C6.9483): 6min/2,5km

If you are interested, i can doing same tests with route with more long distance, other cities, other transports. 😀

**Comment by biodranik** [comment]
See also #1845 and #3072
